pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -69,14 +69,14 @@ import (
 )
 
 const (
-	dockerProcessName     = "dockerd"
-	dockerPidFile         = "/var/run/docker.pid"
-	containerdProcessName = "docker-containerd"
-	containerdPidFile     = "/run/docker/libcontainerd/docker-containerd.pid"
-	maxPidFileLength      = 1 << 10 // 1KB
+	dockerProcessName = "dockerd"
+	dockerPidFile     = "/var/run/docker.pid"
+	containerdPidFile = "/run/docker/libcontainerd/docker-containerd.pid"
```

Comment:
is this still the location of the pid file?

Reply:
AFAIK，the pid file is removed. however， I think we should check it for backward compatible.